packages/server-wallet/src/protocols/__test__/application.test.ts

```diff
@@ -16,13 +16,13 @@ const outcome2 = simpleEthAllocation([
   {amount: BN.from(5), destination: bob().destination},
   {amount: BN.from(5), destination: alice().destination},
 ]);
-const prefundState = {outcome, turnNum: 0};
-const prefundState2 = {outcome: outcome2, turnNum: 0};
-const postFundState = {outcome, turnNum: 2};
-const closingState = {outcome, turnNum: 4, isFinal: true};
+const prefundState = {outcome, turnNum: 0, participants: [alice(), bob()]};
+const prefundState2 = {outcome: outcome2, turnNum: 0, participants: [alice(), bob()]};
+const postFundState = {outcome, turnNum: 2, participants: [alice(), bob()]};
+const closingState = {outcome, turnNum: 4, isFinal: true, participants: [alice(), bob()]};
 
-const runningState = {outcome, turnNum: 7};
-const closingState2 = {outcome, turnNum: 8, isFinal: true};
+const runningState = {outcome, turnNum: 7, participants: [alice(), bob()]};
+const closingState2 = {outcome, turnNum: 8, isFinal: true, participants: [alice(), bob()]};
```

Comment:
This suggestion would go with the above one.
```suggestion
const runningState = {outcome, turnNum: 7, participants};
const closingState2 = {outcome, turnNum: 8, isFinal: true, participants};
```